### PR TITLE
Providing some useful information on how to provide classes to blocks…

### PR DESCRIPTION
**Pull request by eduard13**
… and containers

## Purpose of this pull request

This pull request (PR) improves the `Common layout customization tasks` by providing some info on how the css classes can be added to containers and blocks.

## Affected DevDocs pages

- https://devdocs.magento.com/guides/v2.2/frontend-dev-guide/layouts/xml-manage.html
- https://devdocs.magento.com/guides/v2.3/frontend-dev-guide/layouts/xml-manage.html

## Links to Magento source code

-  https://github.com/magento/magento2/blob/2.3/app/code/Magento/Theme/view/frontend/layout/default.xml#L47-L51
- https://github.com/magento/magento2/blob/2.3/app/code/Magento/Theme/view/frontend/layout/default.xml#L34

### CODE REVIEW

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

File: src/guides/v2.3/frontend-dev-guide/layouts/xml-manage.md [hunk 1]
```diff
@@ -188,6 +188,17 @@ To wrap div or block using container see example:
 </container>
 ```
 
+To add few new classes to the container:
```
Comment (jeff-matthews) [inline src/guides/v2.3/frontend-dev-guide/layouts/xml-manage.md]:
```suggestion
To add new classes to the container:
```

File: src/guides/v2.3/frontend-dev-guide/layouts/xml-manage.md [hunk 2]
```diff
@@ -188,6 +188,17 @@ To wrap div or block using container see example:
...
+
+```xml
+<referenceContainer name="page.wrapper" htmlClass="my-new-page-wrapper-class second-class"/>
+```
+
+![Container Classes]({{ site.baseurl }}/common/images/container-classes-result.png)
+
+{:.bs-callout-warning}
+Using this approach, you will override the existing classes.
```
Comment (jeff-matthews) [inline src/guides/v2.3/frontend-dev-guide/layouts/xml-manage.md]:
```suggestion
This method overrides existing classes.
```

File: src/guides/v2.3/frontend-dev-guide/layouts/xml-manage.md [hunk 3]
```diff
@@ -218,6 +229,18 @@ Example: pass the image to the `logo` block.
 </referenceBlock>
 ```
 
+To add new a new class to the block:
```
Comment (jeff-matthews) [inline src/guides/v2.3/frontend-dev-guide/layouts/xml-manage.md]:
```suggestion
To add a new class to the block:
```

**Comment by eduard13** [comment]
Hi @jeff-matthews, thank you for suggestions, have just added them.

**Pull request merged**